package.json

```diff
@@ -32,5 +32,8 @@
     "prettier": "^3.4.2",
     "standard": "^17.1.2",
     "typescript": "^5.7.3"
+  },
+  "dependencies": {
+    "fastify": "^5.2.1"
```

Comment:
This does not look right to me. The production dependencies should be specified in individual sub-packages (e.g. `api/package.json`)

Please revert this block.